### PR TITLE
Pass strength

### PR DESCRIPTION
**Pull request by Matt1994**
The last pull request for this was simply closed without merging. Please do not let this happen again.

### CODE REVIEW

**Review by leonardas103 — COMMENTED** [review]
Refactoring and proper spacing much appreciated.

File: views/create/create.handlebars [hunk 1]
```diff
@@ -67,18 +67,18 @@
...
-						<div id='seperator' class='fieldSeperator card'>
-							<div class='card-header'>
-								<h5>Fields not on the card:</h5>
-								<small>Unlimited.</small>
-							</div>
-						</div>
+                        <div id='seperator' class='fieldSeperator card'>
+                            <div class='card-header'>
+                                <h5>Fields not on the card:</h5>
+                                <small>Unlimited.</small>
```
Comment (leonardas103) [inline views/create/create.handlebars]: Do you need to explicitly say 'unlimited'? I think you leave it out or make it in a tooltip.
Also maybe "▼ Fields not on the card" looks nicer with the arrow.

File: views/login.handlebars [hunk 2]
```diff
@@ -56,8 +56,8 @@
                         <label for="password" class="cols-sm-2 control-label">Password</label>
                         <div class="cols-sm-10">
                             <div class="input-group">
-                                <span class="input-group-addon"><i class="fa fa-lock fa-lg" aria-hidden="true"></i></span>
-                                <input type="password" class="form-control" name="password" id="password" placeholder="Enter your password" />
+                                <span class="input-group-addon" data-toggle="tooltip" title="A strong password contains at least 8 characters, 1 number, 1 capital letter and 1 symbol"><i class="fa fa-lock fa-lg" aria-hidden="true"></i></span>
```
Comment (leonardas103) [inline views/login.handlebars:59]: I think this tooltip should be moved to the bottom. So have like: Strength<sup>?</sup> and when you hover over the question mark, it shows the message. 

Also I suggest the following modification:
Very Weak: length < 8
Weak: >8 && lowercase only
Fair: >8 && lowercase && (upper || symbom || num)
Strong: >8 && lowercase && upper && (symbol || num)
Very Strong: >8 lowercase && upper && symbol && num

**Review by leonardas103 — COMMENTED** [review]
Apart from the two thing mentioned, I think the "| Home" addition blends too much with the Medical Id text, maybe an extra space or a different font is needed.

**Comment by Theys96** [comment]
Honestly, the password bar turning red alarms me and I think users who don't know it's the password strength indicator is wrong will think something is actually wrong.

**Comment by Matt1994** [comment]
That's what Peter requested. We brought this up to him at the meeting but he still wanted to go with it

**Comment by Theys96** [comment]
Page title thing is clever! Could you make it appear like that in the title tag as well?

**Comment by Matt1994** [comment]
Already done

File: views/login.handlebars (hunk 2 above)
Comment (Matt1994) [reply views/login.handlebars:59]: I'm gonna include the Very Weak condition in it, but other than that I believe it to be fine

File: views/create/create.handlebars (hunk 1 above)
Comment (Matt1994) [reply views/create/create.handlebars]: I'll remove unlimited, but I think the arrow is unnecessary

**Review by Theys96 — APPROVED** [review]
Looks good.

**Pull request merged**